src/pages/search/SearchFilter.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Dropdown, DropdownToggle, DropdownMenu } from 'reactstrap';
+
+interface Props {
+  children: React.ReactNode;
+  label: String;
+}
+
+const SearchFilter = ({ children, label }: Props) => {
+  const [isOpen, setOpen] = React.useState<boolean>(false);
+  return (
+    <Dropdown isOpen={isOpen} toggle={() => setOpen(!isOpen)}>
```

Comment:
do you know how easy it is to drop in form items like check boxes and stuff like that in a dropdown?  I havent tried it yet but I was also looking at the `<Collapse />` component for the filter menus that might be easy to use

Reply:
`Dropdown` seems to accept arbitrary content reasonably well. I kinda like the `Collapse` approach from a UX perspective though, it makes room for the form filters without obscuring any other content.

In any event, I'll be hiding these controls before this PR is over (issue says "Ignore filters for now" which I have already failed to do) but let's try to keep that in mind when we do get there